tpl/tplimpl/embedded/templates/helpers/plain_title.html

```diff
@@ -0,0 +1 @@
+{{- .Title | $.RenderString | plainify -}}
```

Comment:
As suggested earlier, I'd avoid introducing `.RenderString` in this PR.

```suggestion
{{ .Title  | plainify -}}

```

(Also, consider not putting this under `helpers`.)

Reply:
Hmm, was thinking. Maybe this should just be named `title.html`.

Reply:
See above and [the docs](https://gohugo.io/functions/plainify).